src/extension/options-page/DashboardDialogs/Feedback.tsx

```diff
@@ -25,13 +25,15 @@ export function DashboardFeedbackDialog(props: WrappedDialogProps) {
                             value={name}
                             onChange={(e) => setName(e.target.value)}
                         />
-                        <TextField
+                        {/* Todo: fix the whole feedback loop asap, now just mailto info@dimension.im */}
+                        {/* <TextField
```

Comment:
```suggestion
                        {/* <TextField
```

why not use `hidden` attribute